c10/util/Half.h

```diff
@@ -448,6 +448,17 @@ struct alignas(4) complex<Half> {
     imag_ = a * d + b * c;
     return *this;
   }
+
+  C10_HOST_DEVICE complex<Half>& operator/=(const complex<Half>& other) {
```

Comment:
Copied from
https://github.com/pytorch/pytorch/blob/a0cc38eaca46e25f561adf9d5b5655f625c39377/c10/util/complex.h#L247-L258

Reply:
maybe just use opmath_t for consistency?

Reply:
`opmath_t` lives in `ATen`. AFAIK, `c10` should not include anything from `ATen`.